### ISSUE TITLE
Source-compiled 0.9.0 version performs worse than binary-installed one

### ISSUE THREAD

**Issue by chhwang** [tensorflow/tensorflow#3675]
I'm running my ResNet-32 benchmark model. My code looks quite similar with the code in TensorFlow GitHub:

https://github.com/tensorflow/tensorflow/tree/master/tensorflow/models/image/cifar10

but the model is changed into my own ResNet-32 implementation. 

I'm measuring how long it takes to train a minibatch with size 128, and here are brief results. I used one GPU, 64bit Ubuntu 14.04 and Python 2.7 for all experiments.

| TensorFlow Version           | CUDA/cuDNN Version | GPU              | Elapsed Time (ms)  |
| :------------------------: | :---------------------: | :-----------: | :------------------: |
| 0.9.0 (pip installed)           |             7.5 / 4.0.7           | GTX Titan X |              75                |
| 0.9.0 (745f16f790d2f9abf2c6a70d6b383b152cca3cff)  |         8.0 RC / 5.0.5        |   GTX 1080   |             330              |
| 0.8.0 (ea9e00a630f91a459dd5858cb22e8cd1a666ba4e)           |         8.0 RC / 5.0.5        |   GTX 1080   |             60                |

The 0.8.0 version commit contains the first support for CUDA 8.0 RC as far as I know and I found this to use my GTX 1080 without performance drop. Now I can use it with reasonable performance, but I want to figure out why the 0.9.0 compiled version is 4 times slower than before.

**Comment by ppwwyyxx** [comment]
I found that batch norm becomes very slow recently (#3676). Could you try removing batchnorm and benchmark again?

**Comment by chhwang** [comment]
@ppwwyyxx I just tried it and I think you're right. If I remove all batch norm layers, the elapsed time becomes 330 -> 38 ms with 745f16f790d2f9abf2c6a70d6b383b152cca3cff. The same experiment done with pip installed 0.9.0 takes 34 ms.

**Comment by poxvoculi** [comment]
Marking as duplicate of #3676
